polyfill/spatial-navigation-polyfill.js

```diff
@@ -223,7 +227,7 @@
       if (!createSpatNavEvents('beforefocus', bestCandidate, null, dir))
         return true;
 
-      bestCandidate.focus();
+      bestCandidate.focus({preventScroll:true});
```

Comment:
If the element was outside of current visible area, then what is happened?
can user sees it?

Reply:
Oh, yes this is my mistake. For the  spatial-navigation-action:focus case, it shouldn't prevent scroll.
I'll remove this line. Thank you!

Reply:
Fix this via: https://github.com/WICG/spatial-navigation/pull/202